keyboards/polykybd/tools/status_oled_preview.py

```
@@ -74,12 +81,13 @@ def bundle(name):
 
     disp = bundle('NotoSans_Regular11pt7b')
     small = bundle('NotoSans_Medium8pt7b')
+    tiny = bundle('NotoSans_Regular_Tiny_6pt7b')   # language index only (see status_oled.c)
     icons = None
     for n, f in F.items():
```

Ruff (0.16.0)

[warning] Loop control variable `n` not used within loop body

Rename unused `n` to `_n`

(B007)